packages/menu/src/MenuItem/MenuItem.styles.ts

```diff
@@ -9,12 +9,13 @@ import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
 import { palette } from '@leafygreen-ui/palette';
 import { color, spacing } from '@leafygreen-ui/tokens';
 
-import { LGIDs } from '../constants';
 import { menuColor } from '../styles';
+import { getLgIds } from '../utils';
 
 import { Variant } from './MenuItem.types';
+const lgIds = getLgIds();
```

Comment:
This'll be different if a custom lgid is provided. technically I guess it doesn't matter here though